dask/base.py

```diff
@@ -765,7 +774,33 @@ def label(x):
     elif color:
         raise NotImplementedError("Unknown value color=%s" % color)
 
-    return dot_graph(dsk, filename=filename, **kwargs)
+    # Determine which engine to dispatch to, first checking the kwarg, then config,
+    # then whichever of graphviz or ipycytoscape are installed, in that order.
+    engine = engine or config.get("visualization.engine", None)
+
+    if not engine:
+        try:
+            import graphviz  # noqa: F401
+
+            engine = "graphviz"
+        except ImportError:
+            try:
+                import ipycytoscape  # noqa: F401
+
+                engine = "cytoscape"
+            except ImportError:
+                pass
+
+    if engine == "graphviz":
+        from dask.dot import dot_graph
+
+        return dot_graph(dsk, filename=filename, **kwargs)
+    elif engine in ("cytoscape", "ipycytoscape"):
+        from dask.dot import cytoscape_graph
+
+        return cytoscape_graph(dsk, filename=filename, **kwargs)
+    else:
+        raise ValueError(f"Visualizer {engine} not recognized")
```

Comment:
Now when neither `graphviz` nor `ipycytoscape` are installed we get this error message:

```python
ValueError: Visualizer None not recognized
```

We should probably have a `if engine is None` check and raise an error about needing to install `graphviz` or `ipycytoscape`

Reply:
Makes sense, fixed